tobago-example/tobago-example-demo/src/main/webapp/content/900-test/2300-link/Link.xhtml

```diff
@@ -19,13 +19,19 @@
 <ui:composition template="/main.xhtml"
                 xmlns="http://www.w3.org/1999/xhtml"
                 xmlns:tc="http://myfaces.apache.org/tobago/component"
-                xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
-                xmlns:f="http://xmlns.jcp.org/jsf/core">
+                xmlns:ui="http://xmlns.jcp.org/jsf/facelets">
   <tc:link id="aLink" label="a.link" link="https://www.apache.org/">
     <tc:style width="200px"/>
   </tc:link>
   <br/>
   <tc:link id="buttonLink" label="button.link">
     <tc:style width="200px"/>
   </tc:link>
+
+  <tc:link id="dropdownRepeat" label="Dropdown with ui:repeat" omit="true">
+    <tc:style customClass="icon-button"/>
+    <ui:repeat var="river" value="#{forEachController.rivers}">
+      <tc:link id="#{river.name}" label="#{river.name}" omit="true"/>
```

Comment:
id="#{river.name}" has to be removed, to avoid a IllegalArgumentException: "component identifier must not be a zero-length String".

In Tobago 6 the river.name-id of the tc:link is also ignored but instead of an empty string a generated id is set. This seems to be another issue.